app/config/services/platform.xml

```diff
@@ -229,7 +229,17 @@
             <tag name="security.voter"/>
         </service>
 
-        <!-- Citizen projectss -->
+        <!-- Citizen projects -->
+        <service id="app.citizen_project.manager" class="AppBundle\CitizenProject\CitizenProjectManager">
+            <argument type="service" id="doctrine"/>
+            <call method="setStorage">
+                <argument type="service" id="app.storage" />
+            </call>
+            <call method="setGlide">
+                <argument type="service" id="app.glide" />
+            </call>
+        </service>
+
```

Comment:
you should inject services via constructor and declare an alias to use this service with its FQCN.

`<service alias="app.citizen_project.manager" id="AppBundle\CitizenProject\CitizenProjectManager" />`

Reply:
`CitizenProjectManager` is already defined as a service in `services.xml`. 
@Remg The alias is not needed IMO. Just use the FQCN.

Reply:
autowiring fails tho